nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -312,6 +340,10 @@ def do_compression(
         self._set_weight_compression_config(ratio_defining_params, model, graph, activations)
         nncf_logger.info(self._get_bitwidth_distribution_str(all_weight_params, ratio_defining_params))
 
+        if self._awq and activations is not None and self._mode != CompressWeightsMode.NF4:
```

Comment:
Do we need an option for calling AWQ before mixed precision or after?
@AlexKoff88

Reply:
Let's go with this flow for now.